src/search.cpp

```diff
@@ -1450,10 +1451,9 @@ Value Search::Worker::search(
     // Adjust correction history if the best move is not a capture
     // and the error direction matches whether we are above/below bounds.
     if (!ss->inCheck && !(bestMove && pos.capture(bestMove))
-        && (bestValue < ss->staticEval) == !bestMove)
+        && (bestValue > ss->staticEval) == bool(bestMove))
```

Comment:
Shouldn't this be
```suggestion
        && (bestValue >= ss->staticEval) == bool(bestMove))
```
to be non-functional?

Reply:
when `bestValue == ss->staticEval` the bonus is 0, so it's essentially a no-op